modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultCodegen.java

```diff
@@ -2310,6 +2417,17 @@ public Schema unaliasSchema(Schema schema) {
         return ModelUtils.unaliasSchema(this.openAPI, schema, schemaMapping);
     }
 
+    /**
+     * Return the name of a variant schema
+     *
+     * @param names          List of names
+     * @return name of the oneOf schema
+     */
+    @SuppressWarnings("static-method")
+    public String toVariantName(List<String> names) {
+        return "oneOf<" + String.join(",", names) + ">";
```

Comment:
i think this is for a schema covering all the schema specified in the responses (e.g. 2xx, 3xx, 4xx).

should this be `anyOf` instead as technically the payload in the response can technically match more than one schema defined in the responess?